support-workers/src/test/scala/com/gu/support/workers/createZuoraSubscription/CreateZuoraSubscriptionSupporterPlusStepsSpec.scala

```diff
@@ -0,0 +1,174 @@
+package com.gu.support.workers.createZuoraSubscription
```

Comment:
this whole test is based on the above one, which I've moved both into a new package together to group them nicely